src/Logging.AzureAppServices/src/BlobLoggerProvider.cs

```diff
@@ -55,15 +56,25 @@ public BlobLoggerProvider(IOptionsMonitor<AzureBlobLoggerOptions> options)
             _fileName = value.ApplicationInstanceId + "_" + value.BlobName;
             _blobReferenceFactory = blobReferenceFactory;
             _httpClient = new HttpClient();
+            _customPrefixFileName = value.CustomPrefixFileName;
         }
 
         internal override async Task WriteMessagesAsync(IEnumerable<LogMessage> messages, CancellationToken cancellationToken)
         {
             var eventGroups = messages.GroupBy(GetBlobKey);
+            DateTime currDate = DateTime.Now;
             foreach (var eventGroup in eventGroups)
             {
                 var key = eventGroup.Key;
-                var blobName = $"{_appName}/{key.Year}/{key.Month:00}/{key.Day:00}/{key.Hour:00}/{_fileName}";
+                string blobName;
+
+                if (!string.IsNullOrEmpty(_customPrefixFileName))
+                {
+                    string filename = $"{_customPrefixFileName}{currDate.Year}{currDate.Month:00}{currDate.Day:00}.txt";
+                    blobName = $"{_appName}/{filename}";
+                }
+                else
+                    blobName = $"{_appName}/{key.Year}/{key.Month:00}/{key.Day:00}/{key.Hour:00}/{_fileName}";
```

Comment:
```suggestion
                else
                {
                    blobName = $"{_appName}/{key.Year}/{key.Month:00}/{key.Day:00}/{key.Hour:00}/{_fileName}";
                }
```

nit, this is the style we use in this repo.